nginx/sites-enabled/hearts

```diff
@@ -0,0 +1,13 @@
+server {
+    listen 80;
+    server_name lonelyhearts.cards;
+    charset utf-8;
+    root /app;
+
+    location / {
+        proxy_pass http://web:5000;
```

Comment:
Tells nginx to forward requests on port 80 to the web container, port 5000 (the container for our flask app)